src/Sylius/Bundle/WebBundle/Resources/views/Frontend/layout.html.twig

```diff
@@ -134,7 +134,7 @@
         {% block footer %}
             <div class="footer">
                 <p class="text-muted">
-                    &copy; Sylius, 2011 - {{ 'now'|date('Y') }}.
+                    &copy; Sylius, 2011 - {{ 'now'|sylius_localized_date('Y') }}.
```

Comment:
Another issue is that if we need to show in calendar-aware fashion we need to convert **start year** too :D.

Any way this is just the sample theme/template I don't it's such an issue to add processing here, or we can just show Gregorian years here.

Hard-coded `2011 - 2015` is a good idea too.

@pjedrzejewski Can decide here!?